src/OSWindow-Core/OSKeySymbols.class.st

```diff
@@ -15,55 +15,29 @@ Class {
 		'OSK_7',
```

Comment:
Ronie why only some are kept?

Reply:
I moved the generation and setting of these constants into a literal in the data method. I removed the key symbols that do not have proper mapping into a KeyboardKey and where I could not find a corresponding X11 key code to introduce a mapping for them. This is to ensure that all of the named symbols have a corresponding mapping in KeyboardKey. As for the removed symbols in the end, there are either: rare to be present, or rarely handled by an end-user application (e.g.: the Power button for shutting down the machine).

Reply:
Could you add this kind of comment in the class?
Thanks.